src/test/java/seedu/guilttrip/logic/commands/FindExpenseCommandTest.java

```diff
@@ -84,20 +85,6 @@ public void execute_multipleKeywords_multipleExpensesFound() {
                 model.getFilteredExpenses());
     }*/
 
-    /*@Test
-    public void execute_multipleKeywordsMultiplePredicates_singleExpensesFound() {
-        CommandHistory chs = new CommandHistoryStub();
-        String expectedMessage = String.format(MESSAGE_ENTRIES_LISTED_OVERVIEW, model.getFilteredExpenses().size() - 2);
-        EntryContainsDescriptionPredicate predicate = preparePredicate("mala cotton Mars");
-        EntryContainsAmountPredicate amtpredicate = new EntryContainsAmountPredicate(500);
-        List<Predicate<Entry>> predicateToPassIn = Arrays.asList(predicate, amtpredicate);
-        FindExpenseCommand command = new FindExpenseCommand(predicateToPassIn);
-        expectedModel.updateFilteredExpenses(predicate.and(amtpredicate));
-        assertCommandSuccess(command, model, expectedMessage, expectedModel, chs);
-        assertEquals(Arrays.asList(TypicalEntries.TRAVEL_EXPENSE),
-                model.getFilteredExpenses());
-    }*/
-
```

Comment:
Any idea why this test failed?

Reply:
Works now.

Reply:
Do you mind bringing this test back? thanks.